randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -210,7 +210,15 @@ Extra - room_id: [6]
   * L1 Hatch to Entrance Lobby/Door to Owtch Cache A
   * Extra - door_idx: (14,)
   > Pickup (Missile Tank)
-      Can Use Any Bombs
+      Any of the following:
+          Can Use Power Bombs
+          All of the following:
+              Can Use Bombs
```

Comment:
just checking, but you can't hit the bomb block my just doing a double bomb jump right?

Reply:
Double bomb jumps dont work in fusion

Reply:
A single bomb jump doesn't work either.